azure-pipelines.yml

```diff
@@ -0,0 +1,20 @@
+trigger:
+  - master
+
+pr:
+  - master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '10.x'
+    displayName: 'Install Node.js'
+
+  - script: |
+      npm install
```

Comment:
I'm missing the `npm test` step here

Reply:
I was removing it on purpose to check it out something, but please, try to put it back, you'd see an error as if the dependencies werent installed